ncm-spma/src/main/perl/spma/yum.pm

```diff
@@ -66,6 +67,7 @@ use constant YUM_CMD => qw(yum -y shell);
 use constant YUM_DISTRO_SYNC => qw(yum -y distro-sync);
 
 use constant VALID_PACKAGE_NAME => qr{^([\w\.\-\+]+)[*?]?};
+use constant REPO_INCLUDE => 1;
```

Comment:
Use Readonly for newly introduced constants?

Reply:
i prefer not to mix those, the inheritance is also a bit different, so i prefer not to